Makefile

```diff
@@ -39,8 +42,16 @@ all: test build
 
 test: clean cover.out
 
-unit:
-	go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
+unit: kubebuilder
+	$(KUBEBUILDER_ENV) go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
+
+# Install kubebuilder if not found
+KUBEBUILDER_DIR := /tmp/kubebuilder_2.3.1_$(OS)_$(ARCH)/bin 
+kubebuilder:
+ifeq (, $(shell which kubebuilder))
```

Comment:
personally I manage kubebuilder with `brew`. is it better ux to just log a message asking you to install it?

Reply:
probably, but we want this to run automatically and it's not clear to me that openshift-ci runs unit tests in an image that we can control the build of (the `src` image). If we do, then I'm all for that route instead.

Reply:
We control the base image there - see `base.Dockerfile`. It would be great to not download this every time and be dependent on the network

Reply:
sure, I can add it there.

Reply:
done